### PR TITLE
Market creation optimization

### PR DESCRIPTION
**Pull request by grujawork**

### CODE REVIEW

File: contracts/SportMarkets/Rundown/GamesPlayerProps.sol [hunk 1]
```diff
@@ -313,7 +313,6 @@ contract GamesPlayerProps is Initializable, ProxyOwned, ProxyPausable {
             _player.gameId,
             _append(_player), // gameLabel
             consumer.getGameCreatedById(_player.gameId).startTime, //maturity
-            0, //initialMint
```
Comment (danijelthales) [inline contracts/SportMarkets/Rundown/GamesPlayerProps.sol:316]: I would leave this to not have to release all these contracts

File: contracts/SportMarkets/Rundown/TherundownConsumer.sol [hunk 2]
```diff
@@ -504,7 +504,6 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
                 _gameId,
                 string(abi.encodePacked(game.homeTeam, " vs ", game.awayTeam)), // gameLabel
                 game.startTime, //maturity
-                0, //initialMint
```
Comment (danijelthales) [inline contracts/SportMarkets/Rundown/TherundownConsumer.sol:507]: I would leave this to not have to release all these contracts

File: contracts/SportMarkets/SportPositions/SportPositionalMarket.sol [hunk 3]
```diff
@@ -39,13 +40,10 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
 
     struct SportPositionalMarketParameters {
         address owner;
-        IERC20 sUSD;
         address creator;
         bytes32 gameId;
         string gameLabel;
         uint[2] times; // [maturity, expiry]
-        uint deposit; // sUSD deposit
```
Comment (danijelthales) [inline contracts/SportMarkets/SportPositions/SportPositionalMarket.sol:47]: I would leave initialMint to not have to release all related contracts, even though its not used.

File: contracts/SportMarkets/Rundown/GamesOddsObtainer.sol [hunk 4]
```diff
@@ -487,7 +487,6 @@ contract GamesOddsObtainer is Initializable, ProxyOwned, ProxyPausable {
             _gameId,
             _append(_gameId, _isSpread, _spreadHome, _totalOver), // gameLabel
             consumer.getGameCreatedById(_gameId).startTime, //maturity
-            0, //initialMint
```
[the remark this answers is not in the document]
Comment (grujawork) [reply contracts/SportMarkets/Rundown/GamesOddsObtainer.sol]: done.

File: contracts/SportMarkets/Rundown/GamesPlayerProps.sol (hunk 1 above)
Comment (grujawork) [reply contracts/SportMarkets/Rundown/GamesPlayerProps.sol]: done.

File: contracts/SportMarkets/Rundown/TherundownConsumer.sol (hunk 2 above)
Comment (grujawork) [reply contracts/SportMarkets/Rundown/TherundownConsumer.sol]: done.

**Review by vpoklopic — DISMISSED** [review]

**Review by danijelthales — DISMISSED** [review]

**Review by kirilaa — APPROVED** [review]

**Review by vpoklopic — APPROVED** [review]

**Pull request merged**